docs/en/enterprise-edition/content-collections/administration/action-plans.adoc

```diff
@@ -0,0 +1,63 @@
+== Prisma Cloud Action Plans 
+
+Action Plans programmatically deliver effective remediation steps that are tailored to maximize risk reduction while requiring you to take minimal actions, thereby enhancing efficiency and efficacy in securing assets.
+
+* *Prioritization*: Action Plans group together alerts and assets that can be secured through a single fix, ensuring your security team spends time on the most effective outcomes. Actions are prioritize based on security context and Prisma Cloud alerts. 
+
+* *Execution and Delegation*: Leverage your integrations on Prisma Cloud to help delegate security fixes to your team through Jira tickets and/or Slack messages with a single click.
+
+* *Detailed Visibility*: Ensure that every alert resolved or asset impacted is visible in one location, and provide detailed context to your security teams.
+
+* *Security Fix Efficiency*: Using machine learning and generative models, Action Plans help summarize tasks across the various alerts that impact the same asset, ensuring that there is a comprehensive plan to reduce alerts with the least number of required steps.
+
+
+=== Action Plan Menu
+
+Action Plans seamlessly integrate into existing workflows in Prisma Cloud and do not impact any existing experiences on the platform as shown below. 
+
+image::administration/action-plans-1.gif[]
+
+Follow the steps below to access and explore the Action Plans menu:
+
+. Navigate to *Home > Cloud Security > Action Plans* to access the menu.
+
+. The *Action Plans* navigation menu includes the following three out-of-the-box views:
+
+.. *Overview*— Provides a high-level view of all available *Action Plans*. 
+
+... Select *Add Filter* to filter plans by— Status, Finding Type, and Assignee.
+... Click on the *Sort* tab to sort plans by— Criticality, Alert and Asset Count. Criticality is selected by default. It takes into account not only the number of affected parts of your system, but also security data in the context of your business. 
```

Comment:
Click on the Sort tab to sort plans by— Criticality, Alert Count, or Asset Count.